app/scripts/modules/amazon/loadBalancer/configure/createLoadBalancer.controller.js

```diff
@@ -235,6 +235,24 @@ module.exports = angular.module('spinnaker.loadBalancer.aws.create.controller',
       $scope.existingSecurityGroupNames = [];
     }
 
+    function certificateIdAsARN(accountId, certificateId) {
+      if (certificateId) {
+        // If they really want to enter the ARN...
+        if (!certificateId.indexOf('arn') > -1) {
```

Comment:
You're right there could be a certificate name with 'arn' somewhere in it. Will set this to === 0.